src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+
+    @Override
+    public List get() {
```

Comment:
```suggestion
    public List<Truck> get() {
```
it's better not to use raw types

Reply:
Yeah, IDEA write me message too. But I don't see. Thanks!